_temp_extension/.eslintignore

```diff
@@ -0,0 +1,5 @@
+node_modules
```

Comment:
These `_temp_extension/*` files should normally not be committed to the repo.

And could be added to the main `.gitignore`.

Reply:
Thanks, I think I fixed it in the new commit...